src/poprox_storage/repositories/articles.py

```diff
@@ -31,6 +32,7 @@ def __init__(self, connection: Connection):
             "articles",
             "entities",
             "mentions",
+            "article_image_associations",
```

Comment:
Not sure about the name of this table, but also not sure we have another join table like this that has established a convention.